src/main/drivers/bus_i2c.h

```diff
@@ -45,6 +45,13 @@ typedef enum I2CDevice {
 #define I2CDEV_COUNT 4
 #endif
 
+// Macro to convert CLI bus number to I2CDevice.
+#define I2C_CFG_TO_DEV(x)   ((x) - 1)
+
+// I2C device address range in 8-bit address mode
```

Comment:
Dah! It's '7-bit' address mode. Apologies; it should have been propagated from my source... 😭